### ISSUE TITLE
Handle polarized image data.

### ISSUE THREAD

**Issue by timstaley** [transientskp/tkp#400]
CF https://redmine.astron.nl/lofar_issuetracker/issues/3293

"This includes being able to fit negative sources."

I don't even... I'll just leave this here for now.

**Comment by jbroderick1** [comment]
Yep....you can get negative Q/U/V depending on the orientation of the polarization position angle.

**Comment by timstaley** [comment]
Sure, I can believe the data side of it (would also apply to difference imaging). I'm just not sure how we start to deal with it. I don't think it's insurmountable but we need to think carefully about what this would break, as things stand, and how to generalise.

**Comment by AntoniaR** [comment]
This is going to be needed for upcoming surveys with LOFAR and MeerKAT. Let's set it as a target for Release 6.0

**Comment by AntoniaR** [comment]
This is an ongoing wish list item but not essential for R7. Moving to the long term milestone.
